### ISSUE TITLE
build.sh not working

### ISSUE THREAD

**Issue by toshi0383** [plu/pxctest#32]
At latest master (0.3.2) build fails. (either via Xcode Cmd+B or ./script/build.sh fails)
I could work-around it by removing Spectre from embedded `Commander.xcodeproj` .
Homebrew fails with same error.

```
$ brew unlink pxctest && brew tap plu/pxctest && brew install pxctest
Unlinking /usr/local/Cellar/pxctest/0.3.1... 0 symlinks removed
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
amazon-ecs-cli                       jmxtrans                             rtv

==> Installing pxctest from plu/pxctest
==> Cloning https://github.com/plu/pxctest.git
Updating /Users/a14786/Library/Caches/Homebrew/pxctest--git
==> Checking out tag 0.3.2
==> ./scripts/build.sh /usr/local/Cellar/pxctest/0.3.2/libexec
Last 15 lines from /Users/a14786/Library/Logs/Homebrew/pxctest/01.build.sh:
<unknown>:0: error: no such file or directory: '/tmp/pxctest-20170331-10773-1wigcy1/Dependencies/Commander/.build/checkouts/Spectre.git--7655155069707042687/Sources/Context.swift'
<unknown>:0: error: no such file or directory: '/tmp/pxctest-20170331-10773-1wigcy1/Dependencies/Commander/.build/checkouts/Spectre.git--7655155069707042687/Sources/Expectation.swift'
<unknown>:0: error: no such file or directory: '/tmp/pxctest-20170331-10773-1wigcy1/Dependencies/Commander/.build/checkouts/Spectre.git--7655155069707042687/Sources/Failure.swift'
<unknown>:0: error: no such file or directory: '/tmp/pxctest-20170331-10773-1wigcy1/Dependencies/Commander/.build/checkouts/Spectre.git--7655155069707042687/Sources/Global.swift'
<unknown>:0: error: no such file or directory: '/tmp/pxctest-20170331-10773-1wigcy1/Dependencies/Commander/.build/checkouts/Spectre.git--7655155069707042687/Sources/GlobalContext.swift'
<unknown>:0: error: no such file or directory: '/tmp/pxctest-20170331-10773-1wigcy1/Dependencies/Commander/.build/checkouts/Spectre.git--7655155069707042687/Sources/Reporter.swift'
<unknown>:0: error: no such file or directory: '/tmp/pxctest-20170331-10773-1wigcy1/Dependencies/Commander/.build/checkouts/Spectre.git--7655155069707042687/Sources/Reporters.swift'
Command /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc failed with exit code 1

** BUILD FAILED **


The following build commands failed:
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
(1 failure)

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/plu/homebrew-pxctest/issues
```

**Comment by plu** [comment]
Which Xcode version do you have?

**Comment by toshi0383** [comment]
Sorry forgot to mention.
Xcode8.2.1 on ElCapitan

**Comment by toshi0383** [comment]
On Xcode8.2.1 on Sierra(10.12.4), I encountered same error.

**Comment by plu** [comment]
Thank you. It's probably related to Xcode 8.2.1, since Travis on 8.3 is green. I will look into it tomorrow.

**Comment by plu** [comment]
Should be fixed on `master` and version `0.3.3`.

**Comment by toshi0383** [comment]
Awesome. Now it works🙌
